wd_helper.au3

```diff
@@ -1410,8 +1410,8 @@ EndFunc   ;==>_WD_UpdateDriver
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
 ;                  - $_WD_ERROR_InvalidValue
 ;                  - $_WD_ERROR_NotFound
-; Author ........: Dan Pollak, mLipok
-; Modified ......: 18/06/2021
+; Author ........: Danp2
+; Modified ......: mLipok
```

Comment:
I had always intended to used the Modified header as a date to track when it was last modified. Is there some AutoIt "standard" that says that Modified should contain the additional authors name?

Reply:
https://www.autoitscript.com/wiki/UDF-spec
```
Modified

This is a list of modifications made. At its most basic this is just a list of users who have made changes, but ideally it includes some information about what they did, in which cases it follows the same form as other multi-value lines, with the username and description of modifications seperated by a dash.

```